### DISCUSSION TITLE
"fixed" filter naming

### DISCUSSION THREAD

**Discussion by gogonzo** [insightsengineering/teal.slice#293] [Polls]
not possible to change the selected value(s) by the user. Once set, this filter will always return the same logical expression.

@lcd2yyz @donyunardi @chlebowa @kartikeyakirar @m7pr

**Comment by chlebowa** [comment]
I like both `static` and `fixed`, with a slight preference for `static`.

**Reply by m7pr** [reply]
the same!

**Comment by gogonzo** [comment]
Since there is a draw between `fixed` and `static`. There is no reason to change `fixed` to `static`
